packages/lab/stories/form-field-next/form-field-next.stories.tsx

```diff
@@ -50,154 +50,166 @@ export const Disabled: ComponentStory<typeof FormFieldNext> = (props) => {
   return (
     <FlowLayout style={{ width: "366px" }}>
       <FormFieldNext disabled {...props}>
-        <Label>Disabled Form Field</Label>
+        <FormLabel>Disabled Form Field</FormLabel>
         <InputNext defaultValue="Primary Input value" />
-        <HelperText>This field has been disabled</HelperText>
+        <FormHelperText>This field has been disabled</FormHelperText>
       </FormFieldNext>
       <FormFieldNext disabled {...props}>
-        <Label>Disabled Form Field</Label>
+        <FormLabel>Disabled Form Field</FormLabel>
         <InputNext variant="secondary" defaultValue="Secondary Input value" />
```

Comment:
I'd say we should stick to only primary version for input. It gets confusing having so many variations of input in FF stories.